src/web3/contracts.ts

```diff
@@ -122,9 +123,10 @@ async function waitForContractKit(tries: number) {
 
 // This code assumes that the account for walletAddress already exists in the Keychain
 // which is a responsibility currently handled by KeychainWallet
-export function* getViemWallet(chain: Chain) {
-  if (viemWallets.has(chain)) {
-    return viemWallets.get(chain) as ViemWallet
+export function* getViemWallet(chain: Chain, useValoraTransport?: boolean) {
```

Comment:
Should this function take an object instead of multiple parameters to avoid accidentally setting the wrong values?

Reply:
I left a comment above about this in general 😄 this will be in a follow up